src/components/PeopleTable.tsx

```diff
@@ -1,52 +1,139 @@
-export const PeopleTable = () => {
+import classNames from 'classnames';
+import { useParams, useSearchParams } from 'react-router-dom';
+
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
```

Comment:
tests failed because here issue with imports.
![Screenshot from 2023-10-18 11-59-23](https://github.com/mate-academy/react_people-table-advanced/assets/33029864/18bfd613-f0dd-487c-aef6-7593456b20ec)
![Screenshot from 2023-10-18 11-59-13](https://github.com/mate-academy/react_people-table-advanced/assets/33029864/80d6bc28-4b09-4f6d-9a79-f358bd11bf8b)
![Screenshot from 2023-10-18 12-00-54](https://github.com/mate-academy/react_people-table-advanced/assets/33029864/a6f16c45-5078-462d-9f34-936c12351077)